web/src/features/hostdashboard/HostCalendar.js

```diff
@@ -1,228 +1,238 @@
-                        <p>No accommodations found...</p>
-                    ) :
-                    <div className={calenderStyles.contentContainerCalendar}>
-                        <div className={calenderStyles.calendarHeader}>
-                            <button className={calenderStyles.exportICal} onClick={handleICal}>
-                                Export to calender
-                            </button>
-                        </div>
-                        <div className={calenderStyles.calendarDropdown}>
-                            <div>
-                                <select className={calenderStyles.locationBox}
-                                        onChange={handleSelectAccommodation}>
-                                    <option value="" className={calenderStyles.selectOption}>
-                                        Select your Accommodation
-                                    </option>
-                                    {accommodations.map(accommodation => (
-                                        <option key={accommodation.ID}
-                                                value={accommodation.ID}>
-                                            {accommodation.Title}
-                                        </option>
-                                    ))}
-                                </select>
-                            </div>
-                            {selectedAccommodation !== null && selectedAccommodation !== undefined ? (
-                                <div>
-                                    <p>Booking availability for
-                                        {" " + selectedAccommodation.Title}
-                                    </p>
-                                    <div className={calenderStyles.locationBox}>
-                                        <CalendarComponent passedProp={selectedAccommodation} isNew={false}
-                                                           updateDates={updateDates}
-                                                           componentView={true}
-                                        />
-                                    </div>
-                                </div>
-                            ) : (
-                                <div className={styles.alertMessage}>Please select your Accommodation first</div>
-                            )
-                            }
-                        </div>
-                    </div>}
+    try {
+      const uploadURL = await uploadICalToS3(listOfAccommodations, userId);
+      if (uploadURL) {
+        copyToClipboard(uploadURL);
+      } else {
+        console.error("Failed to POST iCal data");
+      }
+    } catch (error) {
+      console.error("Failed to POST iCal data:", error);
+    }
+  };
+
+  return (
+    <div className="page-body">
+      <h2>Calendar</h2>
+      <div className={styles.dashboardHost}>
+        <Pages />
+        {isLoading ? (
+          <div
+            style={{
```

Comment:
Inline styling weghalen
```suggestion
            style={{
              display: "flex",
              justifyContent: "center",
              alignItems: "center",
              height: "42vh",
              width: "72.5%",
            }}
```